apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/PatientE2EBase.java

```diff
@@ -160,9 +162,23 @@ public void testPatientByIdentifierWhenMbiHashWithBeneDupesExpect404() {
     // bene ids
     List<Object> loadedRecords = loadDataWithAdditionalBeneHistory();
     // history has the same mbi, so returns the same hash;
-    // basically second param doesnt matter for this test
+    // basically second param doesn't matter for this test
     String mbiHash = getMbiHash(currentMbi, false, loadedRecords);
 
+    Map<Optional<String>, List<Beneficiary>> beneficiaryMap =
+        loadedRecords.stream()
+            .filter(Beneficiary.class::isInstance)
+            .map(Beneficiary.class::cast)
+            .toList()
+            .stream()
```

Comment:
Why `toList()` and then immediately `stream()`?

Reply:
Removd this. Not needed